packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -329,46 +336,51 @@ describe('fetch', () => {
       const keys = Object.keys(attributes);
 
       assert.ok(
-        attributes[keys[0]] !== '',
+        attributes[AttributeNames.COMPONENT] !== '',
```

Comment:
I don't want to modify existing tests too much but the order of attributes changed so perhaps using attribute keys is more accurate?

Reply:
Is it possible to use [deepstrictequal](https://nodejs.org/dist/latest-v14.x/docs/api/assert.html#assert_assert_deepstrictequal_actual_expected_message)?

Reply:
It's going to be tricky because of the `http.user_agent` and `http.host` attribute depend on the context of the test runner I believe

Reply:
I've changed it to assert useragent and host separately and use `deepStrictEqual` for the other properties 🙏

Reply:
Reverted the changes to put everything in a new decribe block